Automattic/wp-calypso
# Cancel Purchase: Use the correct currency symbol

#### Changes proposed in this Pull Request

* After a currency change, if the user wants a refund we need to process such refund in the currency they originally paid with. This change will use the refund currency instead of the current currency to display the correct symbol to the user.

#### Testing instructions

* Requires D46082-code
* Purchase any Plan + Domain (Make sure that the domain is paid using the Domain Credit, the cost should be displayed as 0 during checkout)
* Change the currency for that user
* Go to the Purchases Page, select the Plan and click on Cancel.
* The "Cancel Purchase" page should display all the refund currencies in the original currency for that purchase, not the current one.

Before
![image](https://user-images.githubusercontent.com/15204776/86852116-c95b4880-c079-11ea-9fca-304debd5e711.png)

After (Note that the refund amount in the second radio now displays MX$ instead of $)
![image](https://user-images.githubusercontent.com/15204776/86855731-8fda0b80-c080-11ea-89b0-7e483d2dc355.png)
